src/main.rs

```diff
@@ -193,29 +192,25 @@ fn run() -> Result<()> {
 }
 
 fn setup_logging<'a>(getter: &'a impl clap_conf::Getter<'a, String>) {
-    // * Log at info by default.
+    // * Log at `info` by default.
     // * Allow users the option of setting complex logging filters using
-    //   env_logger's `RUST_LOG` environment variable.
-    // * Override both of those if the logging level is set via the `--log`
-    //   command line argument.
-    let env = env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info");
-    let mut builder = env_logger::Builder::from_env(env);
-    builder.format_timestamp(None);
-
-    let log_lvl_from_env = std::env::var_os(env_logger::DEFAULT_FILTER_ENV).is_some();
-
-    if log_lvl_from_env {
-        log::set_max_level(log::LevelFilter::Trace);
-    } else {
-        let level = match getter.grab().arg("log_level").conf("log_level").done() {
-            Some(lvl) => lvl.parse().unwrap(),
-            None => log::LevelFilter::Info,
-        };
-        log::set_max_level(level);
-        builder.filter_level(level);
-    }
+    //   the `RUST_LOG` environment variable.
+    // * Override both of those if the logging level is set via
+    //   the `log_level` config setting.
+
+    let filter = match getter.grab().arg("log_level").conf("log_level").done() {
+        Some(lvl) => tracing_subscriber::EnvFilter::from(lvl),
+        None => tracing_subscriber::EnvFilter::from_default_env()
+            .add_directive(tracing::Level::INFO.into()),
+    };
 
-    builder.init();
+    tracing_subscriber::fmt()
+        .without_time()
+        .with_target(false)
+        .with_env_filter(filter)
+        .compact()
```

Comment:
with compact
![image](https://user-images.githubusercontent.com/1502855/141647919-e6b44792-6fbe-4485-9cc5-a46e0fc64ac4.png)

with full
![image](https://user-images.githubusercontent.com/1502855/141647956-1b33e2c2-3c17-40e6-aafe-451caa224e3e.png)

I prefer compact, looks cleaner imo

Reply:
I'd want to remove the `render:expand:etcetc` things, but that would require a custom formatter, simple to do but maybe not neccessary, this shouldn't be used often